game/scripts/vscripts/units/ai_slime_spawner.lua

```diff
@@ -0,0 +1,56 @@
+function Spawn( kv )
+	if not IsServer() then
+		return
+	end
+
+	if thisEntity == nil then
+		return
+  end
+  thisEntity.bForceKill = false
```

Comment:
This whitespace in this file is insane, go through and fix up the indentation. Using an editor that supports `.editorconfig` will help a lot.